prow/plugins/sigmention/sigmention.go

```diff
@@ -61,18 +60,19 @@ func init() {
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
 	// Only the Description field is specified because this plugin is not triggered with commands and is not configurable.
 	return &pluginhelp.PluginHelp{
```

Comment:
Please populate the `Config` field of PluginHelp since this plugin is now configurable. 
https://github.com/kubernetes/test-infra/blob/744055952280f71c320d01a0ba0c01fcd59499ca/prow/pluginhelp/pluginhelp.go#L36-L39
Since configuration is global, only the "" key needs to be populated.

Reply:
done